src/components/collapsible_nav_beta/collapsible_nav_item/collapsed/collapsed_nav_button.tsx

```diff
@@ -63,6 +64,12 @@ export const EuiCollapsedNavButton: FunctionComponent<
     hideToolTip && tooltipStyles.hidden,
   ];
 
+  const buttonIconAriaLabel = useEuiI18n(
+    'euiCollapsedNavButton.ariaLabelButtonIcon',
+    '"{title}" quick navigation menu',
```

Comment:
I would suggest to remove the `"` here in favor of adding a comma `,`.
I'm not sure if the quotation marks might be read weirdly in some assistive technology (e.g. changes how the title is read or it might be read as "quotation mark") 🤔 
The comma instead will add a pause to the output which results in better separation: "Security [SHORT PAUSE] quick navigation menu [SHORT PAUSE] button"

```suggestion
    '{title}, quick navigation menu',
```

cc @1Copenut

Reply:
I concur with @mgadewoll about removing the quote marks and adding a comma. JAWS in particular is picky about punctuation, and if users have verbosity turned up this could make the announcement longer.

Reply:
I'm just gonna go ahead and commit the suggestion if no objections 👀